README.md

```diff
@@ -142,44 +142,22 @@ command-line arguments:
       my-first-function
     ```
 
-1. Start the built container:
+3. Start the built function container:
 
     ```sh
-    docker run --rm -p 8080:8080 my-first-function
+    docker run --rm -p 8080:8080 -e NODE_ENV=dev my-first-function
     # Output: Serving function...
     ```
 
-1. Send requests to this function using `curl` from another terminal window:
+    > NOTICE: `-e NODE_ENV=dev` is required to display "Serving function...", and you can also append `-e DEBUG=*` to display Express internal debug messages.
+
+4. Send requests to this function using `curl` from another terminal window:
 
     ```sh
     curl localhost:8080
     # Output: Hello, World!
     ```
 
-## Run your function on serverless platforms
-
-### Google Cloud Functions
-
-The
-[Node.js 10 runtime on Google Cloud Functions](https://cloud.google.com/functions/docs/concepts/nodejs-10-runtime)
-is based on the Functions Framework. On Cloud Functions, the Functions Framework
-is completely optional: if you don't add it to your `package.json`, it will be
-installed automatically.
-
-After you've written your function, you can simply deploy it from your local
-machine using the `gcloud` command-line tool.
-[Check out the Cloud Functions quickstart](https://cloud.google.com/functions/docs/quickstart).
-
-### Cloud Run / Cloud Run for Anthos
-
-After you've written your function, added the Functions Framework and updated your `start` script in `package.json`, deploy it to Cloud Run with `gcloud run deploy`. Check out the [Cloud Run quickstart for Node.js](https://cloud.google.com/run/docs/quickstarts/build-and-deploy/nodejs).
-
-If you want even more control over the environment, you can [deploy to Cloud Run for Anthos](https://cloud.google.com/anthos/run/docs/quickstarts/prebuilt-deploy-gke). With Cloud Run for Anthos, you can run your function on a GKE cluster, which gives you additional control over the environment (including use of GPU-based instances, longer timeouts and more).
-
-### Container environments based on Knative
-
-Cloud Run and Cloud Run for Anthos both implement the [Knative Serving API](https://www.knative.dev/docs/). The Functions Framework is designed to be compatible with Knative environments. Just build and deploy your container to a Knative environment.
-
 ## Configure the Functions Framework
 
 You can configure the Functions Framework using command-line flags or
```

Comment:
accepted values: `http` or `event` or `cloudevent`
=> 
accepted values: `http` or `event` or `cloudevent` or `openfunction`?

Reply:
We haven't used signature type to determine openfunction related stuffs yet, will add `openfunction` option later if applicable.